### PR TITLE
feat(dtm): update spec

### PR DESCRIPTION
**Pull request by IronCore864**
PR generated automatically from push-to-fig-autocomplete-action.

### CODE REVIEW

File: src/dtm.ts [hunk 1]
```diff
@@ -243,7 +232,7 @@ const completionSpec: Fig.Spec = {
             {
               name: ["--plugin-dir", "-d"],
               description: "Plugins directory",
-              args: { name: "plugin-dir", default: ".devstream" },
+              args: { name: "plugin-dir" },
```
Comment (SeparateRecords) [inline src/dtm.ts:235]: Should this also have `template: "folders"`?

**Review by SeparateRecords — COMMENTED** [review]
Looks good! Just one question then we should be good to merge 🙂

File: src/dtm.ts (hunk 1 above)
Comment (fedeci) [reply src/dtm.ts:235]: It should be handled in the CLI itself to avoid the same issue in the future

**Review by mschrage — APPROVED** [review]
LGTM! Thanks for contributing @IronCore864. How was the process & documentation around setting up autogenerated specs?

File: src/dtm.ts (hunk 1 above)
Comment (fedeci) [inline src/dtm.ts:235]:
```suggestion
              args: { name: "plugin-dir", template: "folders" },
```

**Pull request merged**